libs/shared/db/mysql/account/src/lib/keysley-types.ts

```diff
@@ -55,8 +55,10 @@ export interface Accounts {
   emailVerified: Generated<number>;
   kA: Buffer;
   wrapWrapKb: Buffer;
+  wrapWrapKb2: Buffer | null;
```

Comment:
nit: I think we should add some slight context on the variables maybe `wrapWrapKbVersion2`

Reply:
Are you thinking this variable should be changed everywhere? (Handlers, db column, etc?)